dockstore-webservice/src/main/resources/swagger.yaml

```diff
@@ -2548,6 +2548,36 @@ paths:
           description: "successful operation"
       security:
       - BEARER: []
+  /entries/{entryId}/versions/{versionId}/descriptionMetrics:
```

Comment:
you don't know need to add swagger for new endpoints (especially if they only use primitive or new types)